README.md

```diff
@@ -37,14 +37,12 @@ flatpak install flathub jp.yvt.OpenSpades
 
 Once installed, you'll be able to launch OpenSpades from inside the desktop menu or from your terminal with `flatpak run jp.yvt.OpenSpades`
 
-#### Building and installing from source
-GCC 4.9 / Clang 3.2 or later is recommended because OpenSpades relies on C++11 features heavily.
-
+#### Building from source
```

Comment:
```suggestion
### On Linux (from source)

#### Building
```

The section structure should look like this:

- On Linux
    - Snap package
    - Flatpak package
- On Linux (from source)
    - Building
    - Installing and launching

Instead of this:

- On Linux
    - Snap package
    - Flatpak package
    - Building from source
    - Installing and launching